src/swarmus_ros_navigation/config/base_global_planner_params.yaml

```diff
@@ -0,0 +1,18 @@
+BaseGlobalPlanner:
+  allow_unknow: True          # Specifies if the unknow grid cells should be use to plan the trajectory
+  default_tolerance: 0.0      # Tolerance aroun the goal
+  
+  visual_potential: false     # Specifies or not to visualize the potential area computed via a PointCloud2
+
+  use_dijkstra: true          # If true, use Dijkstra's algorithm. Otherwise, use A*.
```

Comment:
Its more resource efficient indeed but because of the discrete occupation grid, the found path might not be that optimal. You can see their difference in this wiki:
http://wiki.ros.org/global_planner

However, this is not final and it might change later if we realize that we dont see a big difference between the A* path and an optimal path.